sqlScripts/001-Create-Delivery-Driver.sql

```diff
@@ -0,0 +1,26 @@
+DELIMITER $$
+DROP PROCEDURE IF EXISTS my_transaction $$
+CREATE PROCEDURE my_transaction ()
+BEGIN
+	START TRANSACTION;
+
+
+INSERT INTO DeliveryEmployee (EmployeeID, EmployeeFirstName, EmployeeSurname, Salary, BankAccountNumber, NationalInsurance)
+VALUES ('', 'Jonny', 'Evans', '100000', '01234', 'AB12345', 'NI12345');
```

Comment:
If your `EmployeeID` is an auto increment value you shouldn't need to specify it as being blank here